crates/cheatcodes/src/evm.rs

```diff
@@ -9,7 +10,7 @@ use foundry_common::fs::read_json_file;
 use foundry_evm_core::backend::DatabaseExt;
 use foundry_utils::types::ToAlloy;
 use revm::{
-    primitives::{Account, Bytecode, SpecId, KECCAK_EMPTY},
+    primitives::{Account, Bytecode, HashMap as rHashMap, SpecId, KECCAK_EMPTY},
```

Comment:
Any reason as to why we're not using standard hashmap?

Reply:
I think there was a type conflict before all the migrations maybe? but i changed it and it works fine, good catch!